src/prefect/infrastructure/process.py

```diff
@@ -54,6 +55,10 @@ class Process(Infrastructure):
         default=True,
         description="If set, output will be streamed from the process to local standard output.",
     )
+    cwd: Union[str, bytes, None] = Field(
```

Comment:
Can you do `working_dir` here instead? Just supporting `Optional[str]` here is fine too, users shouldn't ever need to use bytes in this higher-level interface.

Reply:
I'm also tempted to say this should just be `Optional[Path]`, but I'm okay with starting with `str`.

Reply:
👍 We don't need to match the underlying method 1:1, with blocks we need to optimize for the interface in the UI.